lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -201,7 +206,7 @@ namespace Js
         Js::RecyclableObject *handlerObj = this->MarshalHandler(requestContext);
 
         //2. If handler is null, then throw a TypeError exception.
-        if (handlerObj == nullptr)
+        if (IsRevoked(handlerObj))
```

Comment:
I think we need to either leave this as-is, or put this `IsRevoked()` check before the call to `MarshalHandler`. Same for all other `handleObj` checks.